.github/workflows/onPushToMain.yml

```diff
@@ -0,0 +1,14 @@
+name: version, tag and github release
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  release:
+    uses: salesforcecli/github-workflows/.github/workflows/githubRelease.yml@main
```

Comment:
Is there a reason we are using `actions/checkout` version `2` here? ([source](https://github.com/salesforcecli/github-workflows/blob/main/.github/workflows/githubRelease.yml#L9))